knx/describe.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2022 mobilarte.
+// Licensed under the MIT license which can be found in the LICENSE file.
+
+package knx
+
+import (
+	"time"
+
+	"github.com/vapourismo/knx-go/knx/knxnet"
+)
+
+// Describe a single KNXnet/IP server.
+func Describe(Address string, searchTimeout time.Duration) (*knxnet.DescriptionRes, error) {
+	// Uses a UDP socket.
+	socket, err := knxnet.DialTunnel(Address)
+	if err != nil {
+		return nil, err
+	}
+	defer socket.Close()
+
+	addr, _ := socket.LocalAddr()
+
+	req, err := knxnet.NewDescriptionReq(addr)
+	if err != nil {
+		return nil, err
+	}
+
+	if err := socket.Send(req); err != nil {
+		return nil, err
+	}
```

Comment:
I think this should send multiple times, not just once. You most likely need to add a case in the `for` loop below that triggers a resend in a given interval.

Reply:
I tried to find a note about multiple sends in the spec, have not found any. But if `DescriptionReq` is sent using broadcast, to `255.255.255.255.3671`, this implementation will fail because it uses `DialUDP` and not `ListenUDP` in `socket.go`. Do we need a third type of `socket` that can handle broadcast?

Reply:
The spec doesn't have to specify it. UDP does not guarantee a packet receipt. Hence re-sending it can be beneficial to get consistent results. However, I think it is fine for now.